Dockerfile

```diff
@@ -1,32 +1,35 @@
 # syntax=docker/dockerfile:experimental
 
-# Build stage: Install ruby dependencies
-# ===
-FROM ruby:2.5 AS build-site
-WORKDIR /srv
-ADD . .
-RUN bundle install
-RUN bundle exec jekyll build
-
 # Build stage: Install yarn dependencies
 # ===
-FROM node:12-slim AS yarn-dependencies
+FROM node:13-slim AS yarn-dependencies
 WORKDIR /srv
 ADD package.json .
 RUN --mount=type=cache,target=/usr/local/share/.cache/yarn yarn install
 
 # Build stage: Run "yarn run build-js"
 # ===
 FROM yarn-dependencies AS build-js
-ADD . .
+WORKDIR /srv
+COPY src .
 RUN yarn run build-js
 
 # Build stage: Run "yarn run build-css"
 # ===
 FROM yarn-dependencies AS build-css
-ADD . .
+WORKDIR /srv
+COPY src/_sass src/_sass
 RUN yarn run build-css
 
+# Build stage: Run "yarn run build-site"
+# ===
+FROM yarn-dependencies AS build-site
+WORKDIR /srv
+COPY src/ src/
```

Comment:
This sounds like it could simple be `RUN mkdir src`?

Reply:
Why? we need the files in `src` to be able to build, not just an empty folder

Reply:
Yeah, nvm I was being silly.